### DISCUSSION TITLE
Cursor pagination

### DISCUSSION THREAD

**Discussion by ivandokov** [livewire/livewire#3182] [3. Feature ideas]
I'm wondering if there are any plans to support `cursorPaginate()` pagination.
I couldn't find it in the docs but I know the Laravel feature is pretty new so I guess it is not yet supported.

**Comment by ariaieboy** [comment]
yes its a new feature and livewire wont support this feature you can make a PR and add this to the livewire

but keep in mind that cursor pagination is kinda different from normal pagination and you can't have the number type links for your pagination.

I maybe start working on a PR for it this friday But I can not promise that I will definitely add this feature.

**Reply by ariaieboy** [reply]
https://github.com/livewire/livewire/pull/3215
